src/main/java/frc/robot/subsystems/arm/commands/XboxControl.java

```diff
@@ -0,0 +1,28 @@
+package frc.robot.subsystems.arm.commands;
+
+import edu.wpi.first.wpilibj.XboxController;
+import edu.wpi.first.wpilibj2.command.CommandBase;
+import frc.robot.subsystems.arm.Arm;
+
+public class XboxControl extends CommandBase {
+    private final Arm arm;
+    private final XboxController xboxController;
+
+    public XboxControl(Arm arm, XboxController xboxController) {
+        this.arm = arm;
+        this.xboxController = xboxController;
+        addRequirements(arm);
+    }
+
+    @Override
+    public void execute() {
+        arm.setShoulderJointPower(xboxController.getLeftY());
+        arm.setElbowJointPower(xboxController.getRightY());
+    }
+
+    @Override
+    public void end(boolean interrupted) {
+        arm.setShoulderJointPower(0);
+        arm.setElbowJointPower(0);
+    }
+}
```

Comment:
This is a default command. There's no reason to add an `end` method as the command never ends.